diff -ruNd old-r1q2/linux/snd_linux.c r1q2/linux/snd_linux.c
--- old-r1q2/linux/snd_linux.c	2005-02-02 19:24:31.000000000 +0200
+++ r1q2/linux/snd_linux.c	2005-02-02 19:25:05.000000000 +0200
@@ -73,14 +73,14 @@
 	if (ioctl(audio_fd, SNDCTL_DSP_GETCAPS, &caps)==-1)
 	{
 		perror(snddevice->string);
-        Com_Printf("Sound driver too old\n");
+	        Com_Printf("Sound driver too old\n", LOG_CLIENT);
 		close(audio_fd);
 		return 0;
 	}
 
 	if (!(caps & DSP_CAP_TRIGGER) || !(caps & DSP_CAP_MMAP))
 	{
-		Com_Printf("Sorry but your soundcard can't do this\n");
+		Com_Printf("Sorry but your soundcard can't do this\n", LOG_CLIENT);
 		close(audio_fd);
 		return 0;
 	}
@@ -88,7 +88,7 @@
     if (ioctl(audio_fd, SNDCTL_DSP_GETOSPACE, &info)==-1)
     {   
         perror("GETOSPACE");
-		Com_Printf("Um, can't do GETOSPACE?\n");
+		Com_Printf("Um, can't do GETOSPACE?\n", LOG_CLIENT);
 		close(audio_fd);
 		return 0;
     }
@@ -125,7 +125,7 @@
 	if (!dma.buffer)
 	{
 		perror(snddevice->string);
-		Com_Printf("Could not mmap %s\n", snddevice->string);
+		Com_Printf("Could not mmap %s\n", LOG_CLIENT, snddevice->string);
 		close(audio_fd);
 		return 0;
 	}
@@ -137,7 +137,7 @@
     if (rc < 0)
     {
 		perror(snddevice->string);
-        Com_Printf("Could not set %s to stereo=%d", snddevice->string, dma.channels);
+	        Com_Printf("Could not set %s to stereo=%d", LOG_CLIENT, snddevice->string, dma.channels);
 		close(audio_fd);
         return 0;
     }
@@ -150,7 +150,7 @@
     if (rc < 0)
     {
 		perror(snddevice->string);
-        Com_Printf("Could not set %s speed to %d", snddevice->string, dma.speed);
+	        Com_Printf("Could not set %s speed to %d", LOG_CLIENT, snddevice->string, dma.speed);
 		close(audio_fd);
         return 0;
     }
@@ -162,7 +162,7 @@
         if (rc < 0)
 		{
 			perror(snddevice->string);
-			Com_Printf("Could not support 16-bit data.  Try 8-bit.\n");
+			Com_Printf("Could not support 16-bit data.  Try 8-bit.\n", LOG_CLIENT);
 			close(audio_fd);
 			return 0;
 		}
@@ -174,7 +174,7 @@
         if (rc < 0)
 		{
 			perror(snddevice->string);
-			Com_Printf("Could not support 8-bit data.\n");
+			Com_Printf("Could not support 8-bit data.\n", LOG_CLIENT);
 			close(audio_fd);
 			return 0;
 		}
@@ -182,7 +182,7 @@
 	else
 	{
 		perror(snddevice->string);
-		Com_Printf("%d-bit sound not supported.", dma.samplebits);
+		Com_Printf("%d-bit sound not supported.", LOG_CLIENT, dma.samplebits);
 		close(audio_fd);
 		return 0;
 	}
@@ -194,7 +194,7 @@
 	if (rc < 0)
 	{
 		perror(snddevice->string);
-		Com_Printf("Could not toggle.\n");
+		Com_Printf("Could not toggle.\n", LOG_CLIENT);
 		close(audio_fd);
 		return 0;
 	}
@@ -203,7 +203,7 @@
 	if (rc < 0)
 	{
 		perror(snddevice->string);
-		Com_Printf("Could not toggle.\n");
+		Com_Printf("Could not toggle.\n", LOG_CLIENT);
 		close(audio_fd);
 		return 0;
 	}
@@ -225,7 +225,7 @@
 	if (ioctl(audio_fd, SNDCTL_DSP_GETOPTR, &count)==-1)
 	{
 		perror(snddevice->string);
-		Com_Printf("Uh, sound dead.\n");
+		Com_Printf("Uh, sound dead.\n", LOG_CLIENT);
 		close(audio_fd);
 		snd_inited = 0;
 		return 0;
diff -ruNd old-r1q2/linux/vid_menu.c r1q2/linux/vid_menu.c
--- old-r1q2/linux/vid_menu.c	2005-02-02 19:24:31.000000000 +0200
+++ r1q2/linux/vid_menu.c	2005-02-02 19:28:36.000000000 +0200
@@ -82,8 +82,8 @@
 	else
 		s_brightness_slider[0].curvalue = s_brightness_slider[1].curvalue;
 
-	if ( stricmp( vid_ref->string, "soft" ) == 0 ||
-		 stricmp( vid_ref->string, "softx" ) == 0 )
+	if ( Q_stricmp( vid_ref->string, "soft" ) == 0 ||
+		 Q_stricmp( vid_ref->string, "softx" ) == 0 )
 	{
 		float gamma = ( 0.8 - ( slider->curvalue/10.0 - 0.5 ) ) + 0.5;
 
@@ -145,7 +145,7 @@
 		if ( vid_gamma->modified )
 		{
 			vid_ref->modified = true;
-			if ( stricmp( gl_driver->string, "3dfxgl" ) == 0 )
+			if ( Q_stricmp( gl_driver->string, "3dfxgl" ) == 0 )
 			{
 				char envbuffer[1024];
 				float g;
diff -ruNd old-r1q2/linux/vid_so.c r1q2/linux/vid_so.c
--- old-r1q2/linux/vid_so.c	2005-02-02 19:24:31.000000000 +0200
+++ r1q2/linux/vid_so.c	2005-02-02 19:25:05.000000000 +0200
@@ -326,7 +326,7 @@
 	setreuid(getuid(), getuid());
 	setegid(getgid());
 
-	Com_Printf( "------------------------------------\n");
+	Com_Printf( "------------------------------------\n", LOG_CLIENT);
 	reflib_active = true;
 	return true;
 }
@@ -366,10 +366,10 @@
 		{
 			if ( strcmp (vid_ref->string, "soft") == 0 ||
 				strcmp (vid_ref->string, "softx") == 0 ) {
-Com_Printf("Refresh failed\n");
+				Com_Printf("Refresh failed\n", LOG_CLIENT);
 				sw_mode = Cvar_Get( "sw_mode", "0", 0 );
 				if (sw_mode->value != 0) {
-Com_Printf("Trying mode 0\n");
+					Com_Printf("Trying mode 0\n", LOG_CLIENT);
 					Cvar_SetValue("sw_mode", 0);
 					if ( !VID_LoadRefresh( name ) )
 						Com_Error (ERR_FATAL, "Couldn't fall back to software refresh!");
diff -ruNd old-r1q2/server/server.h r1q2/server/server.h
--- old-r1q2/server/server.h	2005-02-02 19:24:31.000000000 +0200
+++ r1q2/server/server.h	2005-02-02 19:27:26.000000000 +0200
@@ -457,7 +457,7 @@
 void Sys_Minimize (void);
 #endif
 
-void Blackhole (netadr_t *from, qboolean isAutomatic, int mask, int method, const char *fmt, ...) __attribute__ ((format (printf, 4, 5)));;
+void Blackhole (netadr_t *from, qboolean isAutomatic, int mask, int method, const char *fmt, ...) __attribute__ ((format (printf, 5, 6)));;
 
 //
 // sv_phys.c
